repo/packages/O/openldap-admin/2/package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "packagingVersion": "3.0",
+  "name": "openldap-admin",
+  "version": "0.3",
+  "scm": "https://github.com/dinkel/docker-phpldapadmin",
+  "maintainer": "support@mesosphere.io",
+  "website": "http://phpldapadmin.sourceforge.net/wiki/index.php/Main_Page",
+  "description": "phpLDAPadmin is a web-based LDAP client. It provides easy, anywhere-accessible, multi-language administration for your LDAP server.\n\nThis DC/OS package is ready to be used alongside the DC/OS `openldap` package. Other LDAP servers may work, but haven't been tested.",
+  "tags": [
+    "ldap",
+    "directory"
+    ],
+  "preInstallNotes": "Open LDAP Admin requires at least 0.1 CPU and 128MB of RAM.\n\nWARNING: OpenLDAP Admin on DCOS is currently EXPERIMENTAL. There may be bugs, incomplete features, incorrect documentation, or other discrepancies.\n\nThis package will automatically connecto to an existing DC/OS `openldap` package through an automated Virtual IP.\n\nThis package is accessible through the DC/OS `Services` tab. If the `external_access` installation option is selected, it will also create an entry in the DC/OS external load balancer for accessing the UI from outside the cluster, by default in port `11389` . Access the UI at `http://your_DCOS_URL/services/openldapadmin`, or if external_access is selected, also at `http://your_public_node_ip:11389`.",
```

Comment:
corrected -- removed vhost (openldap is TCP not HTTP).
Made external connectivity a separate section for clarity.
Made "enabled" default to false.
Added notes on unencrypted connection.